app/src/main/java/com/netmera/netmerafintech/ImpactFintechApp.kt

```diff
@@ -0,0 +1,35 @@
+package com.netmera.netmerafintech
+
+import android.app.Application
+import androidx.startup.AppInitializer
+import com.netmera.NMInitializer
+import com.netmera.Netmera
+import com.netmera.NetmeraConfiguration
+import com.netmera.netmerafintech.utils.FIREBASE_SENDER_ID
+import com.netmera.netmerafintech.utils.NETMERA_API_KEY
+import com.netmera.nmfcm.NMFCMProviderInitializer
+import com.netmera.nmhms.NMHMSProviderInitializer
+
+class ImpactFintechApp: Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        NMInitializer.initializeComponents(this)
+
+        AppInitializer.getInstance(this)
+
+        AppInitializer.getInstance(this).run {
+            initializeComponent(NMFCMProviderInitializer::class.java)
+            initializeComponent(NMHMSProviderInitializer::class.java)
+        }
+
+        val configBuilder = NetmeraConfiguration.Builder()
+
+        configBuilder.apiKey(NETMERA_API_KEY)
+            .firebaseSenderId(FIREBASE_SENDER_ID)
```

Comment:
I see you've added huawei libraries but didn't see huawei id here. Will you do that later?